### DISCUSSION TITLE
How to generate pages inside folders?

### DISCUSSION THREAD

**Discussion by junevm** [nylo-core/nylo#40] [Q&A] [answered]
I want to have all the pages inside `pages/login` folder. I tried `flutter pub run nylo_framework:main make:page login/test_page` but with no luck.

Any way of doing this?

**Comment by agordn52** [comment, marked as answer]
Hi @a0v0,

This feature is now available in [4.x](https://github.com/nylo-core/nylo/releases/tag/v4.0.0) 🚀
